pyrokinetics/equilibrium/equilibrium.py

```diff
@@ -347,8 +350,11 @@ def __init__(
 
         # Check that floats are valid
         psi_lcfs = float(psi_lcfs) * cocos_factors["PSI"] * eq_units["psi"]
-        if psi_lcfs < psi[0]:
-            raise ValueError("psi_lcfs should be greater than psi[0].")
+        print(Ip_sign, psi_lcfs, psi[0])
```

Comment:
Is this needed? Or just a debugging line?

Reply:
Debugging!